AEPAnalytics/Sources/Analytics.swift

```diff
@@ -555,7 +578,7 @@ public class Analytics: NSObject, Extension {
     ///     - trackData: Dictionary containing tracking data
     ///     - timestamp: timestamp to use for tracking
     ///     - Returns a `Dictionary` containing the vars data
-    private func processAnalyticsVars(trackData: [String: Any]?, timestamp: TimeInterval) -> [String: String] {
+    fileprivate func processAnalyticsVars(trackData: [String: Any]?, timestamp: TimeInterval) -> [String: String] {
```

Comment:
Why use `fileprivate` access here? I don't see any other calls to this method except from `AnalyticsBase.track`.